tests/test_PSWFbasis2D.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from unittest import TestCase
+
+from aspire.basis.pswf_2d import PSWFBasis2D
+
+import os.path
+DATA_DIR = os.path.join(os.path.dirname(__file__), 'saved_test_data')
+
+class PSWFBasis2DTestCase(TestCase):
+    def setUp(self):
+        self.basis = PSWFBasis2D(64, 1.0, 1.0, 0)
+
+    def tearDown(self):
+        pass
+
+    def testPSWFBasis2DEvaluate_t(self):
+        images = np.load(os.path.join(DATA_DIR, 'example_data_np_array.npy'))
```

Comment:
Again, these are way too big.

Reply:
Done in commit 2002953. Reduced to the size of 8X8 .